graphdatascience/session/session_sizes.py

```diff
@@ -1,33 +1,20 @@
 from enum import Enum
-from typing import Type
 
 
-class SessionSizeByMemory(Enum):
+class SessionMemory(Enum):
     """
-    Enumeration representing session sizes.
+    Enumeration representing session memory configurations.
     """
 
-    XS = "1GB"
-    S = "2GB"
-    SM = "4GB"
-    M = "8GB"
-    ML = "16GB"
-    L = "32GB"
-    XL = "64GB"
-    XXL = "128GB"
-    X3L = "256GB"
-    X4L = "320GB"
-    X5L = "512GB"
+    _1GB = "1GB"
+    _2GB = "2GB"
+    _4GB = "4GB"
+    _8GB = "8GB"
+    _16GB = "16GB"
+    _32GB = "32GB"
+    _64GB = "64GB"
+    _128GB = "128GB"
+    _256GB = "256GB"
+    _320GB = "320GB"
+    _512GB = "512GB"
     DEFAULT = "8GB"
```

Comment:
why do we even have the default option?

Reply:
I thought it was a usability feature. If the user really doesn't know, then this size is a good starting point.

We can remove it if you do not believe in it.

Reply:
But not in this PR.

Reply:
I think, if we make the size an optional parameter on the get_and_create, than its not needed to have an enum value for it.

But than its not much overhead to have it, so I dont mind.

Reply:
Actually, as it turns out, we cannot have the same value twice in the enum. So DEFAULT is going out now.